src/openms/include/OpenMS/METADATA/MetaInfoInterfaceUtils.h

```diff
@@ -100,6 +102,17 @@ namespace OpenMS
       return common_keys;
     }
 
+    /// function to copy all meta values from one object to another
+    //TODO get a MetaValue list to copy only those that have been set
+    static void copyMetaValues(const MetaInfoInterface& from, MetaInfoInterface& to)
```

Comment:
this could also go into the MetaInfoInterface class as: addMetaValues(...) but fine with me as is

Reply:
yes, thought about it, too. I think I will do it.

Reply:
the function name is also misleading. Should be 'addMetaValues()'...
This is already fixed in a branch of mine somewhere but you can go ahead .. I'll rebase and merge later